src/endpoints/process_models/process_model_controller.ts

```diff
@@ -94,4 +94,13 @@ export class ProcessModelController {
 
     response.status(this.httpCodeSuccessfulNoContentResponse).send();
   }
+
+  public async terminateProcess(request: HttpRequestWithIdentity, response: Response): Promise<void> {
+    const processInstaceId: string = request.params.processInstanceId;
```

Comment:
```suggestion
    const processInstanceId: string = request.params.process_instance_id;
```

REST parameters are always written with underscores.